shared/login/register/select-other-device/index.desktop.js

```diff
@@ -40,11 +34,48 @@ const Row = ({deviceID, name, type, onSelect}) => {
   )
 }
 
-const SelectOtherDevice = ({onBack, devices, onWont, onSelect, canSelectNoDevice}: Props) => (
+const ResetOption = ({showResetLink, setShowResetLink, onReset}) => (
+  <div>
+    <div style={stylesRow} className="deviceRow" onClick={() => setShowResetLink(true)}>
+      <div style={stylesIconName}>
+        <div style={stylesIconContainer}>
+          <Icon
+            style={{...stylesIcon, fontSize: 28, color: globalColors.black_40, marginLeft: 34}}
+            type="iconfont-close"
+          />
+        </div>
+        <Text type="Body">
+          Uh oh - I don't have any of these devices anymore, or I've uninstalled Keybase from all of them.
+        </Text>
+      </div>
+    </div>
+    {showResetLink &&
+      <div style={{...stylesRow, alignItems: 'center', cursor: 'normal'}}>
+        <div style={stylesIconName}>
+          <Text type="BodyPrimaryLink" onClick={onReset}>
+            RESET MY ACCOUNT & START FROM SCRATCH
+          </Text>
+        </div>
+      </div>}
+  </div>
+)
+
+const SelectOtherDevice = ({
+  onBack,
+  devices,
+  onWont,
+  onSelect,
+  canSelectNoDevice,
+  showResetLink,
+  setShowResetLink,
+  onReset,
+}: Props) => (
   <Container style={stylesContainer} onBack={onBack}>
-    <Text type="Header" style={stylesHeader}>Which device would you like to connect with?</Text>
+    <style>{realCSS}</style>
```

Comment:
if we're going to touch this we might as well switch it to use glamorous so we can not have to inline this kind of styling. lemme know if you haven't done this before.
also this is using all these divs instead of boxes and things like that. can we take a quick pass at modernizing this